sysutils/coreutils/Portfile

```diff
@@ -71,7 +72,7 @@ post-destroot {
     }
 
     # Fix symlink to gnucut
```

Comment:
I would say that this comment is now slightly incorrect. Can you update it to say:

`# Fix symlink to gnu cut`